src/app/pages/wishlist/component/wishlist-item/wishlist-item.component.scss

```diff
@@ -0,0 +1,24 @@
+.heart::ng-deep {
+  svg {
+    path {
+      @apply fill-gray-500;
+    }
+  }
+  @apply mt-6;
+}
+
+.heart:hover ::ng-deep {
+  svg {
+    path {
+      @apply fill-red-600;
+    }
+  }
+}
+
+.fill::ng-deep {
+  svg {
+    path {
+      @apply fill-red-600;
+    }
+  }
+}
```

Comment:
do we have this class in template?

Reply:
Never in template, but several times in scss. 
What should I do?